packages/sensors_plus/sensors_plus/android/src/main/kotlin/dev/fluttercommunity/plus/sensors/SensorsPlugin.kt

```diff
@@ -33,29 +33,29 @@ class SensorsPlugin : FlutterPlugin {
 
         accelerometerChannel = EventChannel(messenger, ACCELEROMETER_CHANNEL_NAME)
         accelerationStreamHandler = StreamHandlerImpl(
-                sensorsManager,
-                Sensor.TYPE_ACCELEROMETER
+            sensorsManager,
+            Sensor.TYPE_ACCELEROMETER
         )
         accelerometerChannel.setStreamHandler(accelerationStreamHandler)
 
         userAccelChannel = EventChannel(messenger, USER_ACCELEROMETER_CHANNEL_NAME)
         linearAccelerationStreamHandler = StreamHandlerImpl(
-                sensorsManager,
-                Sensor.TYPE_LINEAR_ACCELERATION
+            sensorsManager,
```

Comment:
I don't think formatting changes shall go in, as they differ from environment to environment. Let's keep it the same as it has been

Reply:
Formatting isn't different as the project has editorconfig file in the root, so formatting should be the same unless the developer has overriding settings in IDE.